docs/troubleshooting-faq/faq.md

```diff
@@ -0,0 +1,61 @@
+# Frequently asked questions
+
+### Why didn't the sample/project I specified when running the [data download script](../getting-started/accessing-resources/getting-access-to-data.md#using-the-download-data-script) download?
+
+First, we recommend using the `--dryrun` flag when running the script to check which files _would_ be downloaded, to confirm that there is nothing wrong with your internet connection and that you are properly [logged into your AWS profile](../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
+
+If you specify a sample or project ID that is not present in the OpenScPCA data release, then nothing will download, but the script will not explicitly fail or warn you about this.
+
+Therefore, you may wish to list all files in the data release S3 bucket to confirm the sample/project files you are trying to download exist.
+To do this, you'll need to use the AWS CLI [in a terminal](../software-platforms/general-tools/using-the-terminal.md) and run `aws s3 ls` to list the contents of a given S3 bucket.
+
+All data release files are stored in the S3 bucket `s3://openscpca-data-release/`, which you can recursively list the contents of as follows:
+
+1. First, find the names of all the data release buckets, which are named by release date in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601):
+    ```bash
+    # List all releases in the overall release bucket
+    # If you're _not_ working on Lightsail for Research, specify your AWS profile name with `--profile openscpca`
+    aws s3 ls s3://openscpca-data-release/ --profile openscpca
```

Comment:
There is a flag for this in the script! It gives a much more user-friendly output:
```
❯ ./download-data.py --list-releases
Available release dates:
2024-03-11
2024-04-06
2024-05-01```